content/en/blog/2021/non-code-contribution.md

```diff
@@ -0,0 +1,46 @@
+
+# How I contribute
+
+For me, contributing to Kubernetes is all about helping other contributors. I love talking to new contributors about why something is the way it is and they come up with some alternatives to doing that task. That thought exercise can be an enormously helpful contribution. That has been a part of what I've done in the community. Asking why something is the way it is can be a potent tool for improving things. I’ve helped with [new contributor workshops](https://github.com/kubernetes/community/blob/master/events/2018/05-contributor-summit/new-contributor-workshop.md), which meant showing up and sharing my knowledge. I’ve helped onboard folks into the Kubernetes GitHub organization. I’ve hosted community meetings.
+
+To be honest, I think the most complicated contributions to Kubernetes I’ve made involved the [community link shortener](https://github.com/kubernetes/k8s.io/blob/master/k8s.io/configmap-nginx.yaml). That consisted of editing a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/). I had to ask for help once because something wasn’t apparent to me, I think, and there was feedback to create more consistency with already created URLs. But, at no point in time was anyone reluctant to help or the feedback negative. It was a genuinely fantastic experience. Plus, I don’t have to remember the YouTube page URL anymore. I type in [yt.k8s.io](https://yt.k8s.io). Speaking of, we have [YouTube, Slack, and other admins](/docs/comms/moderators/) for tools used throughout the community. They all need administration and, more importantly, participants. These things don’t necessarily require coding knowledge but familiarity with the tool that’s in use.
+
+My work now is focusing more on contributing to our [Contributor Marketing](https://github.com/kubernetes/community/tree/master/communication/marketing-team) subproject. The Contributor Marketing sub project's purpose is to get the word out about how to get involved. Writing this article was an output of one of those meetings. Our community does amazing things every day.
+
+None of those things required me to write code. Non-code contributors are an essential part of our community.
+
+# Hop into the Kubernetes community
+
+To join the Kubernetes community, I recommend finding a SIG where your skills align, doing things you want to do in your off time. There's a list of [Kubernetes SIGs](/resources/community-groups/); it’s had over 200 contributors over the years. If you’re not sure where to go, start with [Contributor Experience](https://github.com/kubernetes/community/tree/master/sig-contributor-experience). We’re here to help.
+
+Often, all you have to do to get started as a contributor is show up to a SIG meeting and tell folks how you want to help. Folks will be more than happy to move whatever hurdles are in your way to help you be productive. There will be some hurdles, of course--Kubernetes is a massive project--but we are all in it together.
```

Comment:
being a fly on the wall first before asking what help folks need or how they can help as an individual is a better onboarding approach but this is a comment vs a review since this is about your experience

Reply:
@parispittman Showing up is half the battle. I do like the fly-on-the-wall idea because that's what I've tended to do in the past. I'll make a change real quick.